src/promptflow-evals/tests/evals/e2etests/README.md

```diff
@@ -0,0 +1,37 @@
+# E2E tests logging to the application insights.
+For the regression tracking we are recording test run times and estimate the installation times of promptflow-evals wheel with and without `azure` optional dependency. The table below summarizes the events we are logging.
+
+## Events
+| Event | Description | Logging value | Is recorded |
+|-------|-------------|---------------|-------------|
+| install_promptflow_no_extras | Installation time of the packages without Azure | The number of packages installed and installation time in seconds | N/A |
+| install_promptflow_with_extras | Installation time of the packages with Azure | The number of packages installed and installation time in seconds | N/A |
+| run_e2e_tests_local | Run time tests marked as `localtest`\*. Each test is being logged as `TestClass::test_name`. | The test run time in seconds | No |
+| run_e2e_tests_azure | Run time tests marked as `azuretest`\*. Each test is being logged as `TestClass::test_name`. These tests write tracking information to Azure. | The test run time in seconds | Yes |
+| performance_tests | Run two tests parked as `performance_test` Each test is being logged as `TestClass::test_name`. These tests write tracking information to Azure. | The test run time in seconds | No |

```

Comment:
Let's remove "two". I added one test to performance test category recently.

Reply:
Fixed.